frontend/src/components/RequestSourceInputs.vue

```diff
@@ -28,18 +28,28 @@
         placeholder="Source Rollup"
         required
       />
+      <InputValidationMessage
+        v-if="v$.sourceChain.$error"
+        :message="v$.sourceChain.$errors[0].$message"
+      />
     </div>
     <div class="flex flex-col justify-between">
       <div class="flex flex-row gap-5">
-        <div class="flex-[9_9_0%] flex flex-col items-end">
+        <div class="flex-[9_9_0%] flex flex-col items-start">
+          <!-- Todo: refactor keydown event to a reusable directive -->
           <Input
             v-model="selectedAmount"
             name="amount"
             type="number"
             placeholder="0.00"
             required
+            :valid="isSelectedAmountValid"
+            @keydown="(e) => ['e', 'E', '+', '-'].includes(e.key) && e.preventDefault()"
```

Comment:
If you think it make sense, go for prohibiting all alpha characters. Probably a regex would be best.